zombies/src/main/java/org/phantazm/zombies/map/shop/FlagSource.java

```diff
@@ -0,0 +1,6 @@
+package org.phantazm.zombies.map.shop;
+
+public enum FlagSource {
```

Comment:
do we reasonably see this being used in places other than shops?

Reply:
possibly, for now it's only used in shops

Reply:
well I would make it ShopFlagSource or something to clarify
also if were to expand this enum then the switch statement might not make sense and might have to throw exc